okhttp-sse/src/test/java/okhttp3/internal/sse/EventSourceHttpTest.java

```diff
@@ -108,9 +108,49 @@
     listener.assertFailure("timeout");
   }
 
+  @Test public void retainsAccept() {
+    server.enqueue(new MockResponse().setBody(""
+        + "data: hey\n"
+        + "\n").setHeader("content-type", "text/event-stream"));
+
+    EventSource source = newEventSource("text/plain");
+
+    assertThat(source.request().url().encodedPath()).isEqualTo("/");
+    assertThat(source.request().header("Accept")).isEqualTo("text/plain");
```

Comment:
alternately you can use RecordedRequest to assert the header made it all the way to the server?